src/auro-icon.js

```diff
@@ -103,17 +108,52 @@ class AuroIcon extends AuroElement {
   }
 
   async firstUpdated() {
-    const svg = await this.fetchIcon(this.category, this.name);
-
-    if (svg) {
-      this.svg = svg;
+    if (this.alaska) {
+      this.svg = this.getAlaskaLogo();
     } else {
-      const penDOM = new DOMParser().parseFromString(penguin.svg, 'text/html');
+      const svg = await this.fetchIcon(this.category, this.name);
 
-      this.svg = penDOM.body.firstChild;
+      if (svg) {
+        this.svg = svg;
+      } else {
+        const penDOM = new DOMParser().parseFromString(penguin.svg, 'text/html');
+
+        this.svg = penDOM.body.firstChild;
+      }
     }
   }
 
+  /**
+   * @private  function to fetch the Alaska Logo icon
+   * @returns {dom} DOM ready HTML to be appended
+   */
+  getAlaskaLogo() {
+    return html`
+      <svg viewbox="0 0 300 90" width="100%" height="100%" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" role="img" aria-hidden="true">
```

Comment:
The svg definitely should not be put directly in the js file, but I'm also unsure how to fetch it if I save it directly as a .svg file. Any suggestions regarding this would be welcome.